src/EPR.SubsidiaryBulkUpload.Function/EPR.SubsidiaryBulkUpload.Function.csproj

```diff
@@ -13,6 +13,7 @@
   </PropertyGroup>
   <ItemGroup>
 	  <FrameworkReference Include="Microsoft.AspNetCore.App" />
+	  <None Include="local.settings.json" />
```

Comment:
I don't think you need this - there should already be an item in this .csproj file with local.settings.json

Reply:
Removed